File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test4/CPlus/SubID.cpp
#include <iostream> 

using namespace std;

class SubID {

	private:
  	string SenderFIXTag_;
  	string TargetFIXTag_;
  	string OnBehalfOfFIXTag_;
  	string DeliverToFIXTag_;
 	
 	public: 

  	// default constructor
	SubID () {
		SenderFIXTag_ = "50";
		TargetFIXTag_ = "57";
		OnBehalfOfFIXTag_ = "116";
		DeliverToFIXTag_ = "129";
  	
  	}
  
  	// by parameters
  	SubID (		string SenderFIXTag_p, string TargetFIXTag_p, string OnBehalfOfFIXTag_p, string DeliverToFIXTag_p){
	  
		SenderFIXTag_ = SenderFIXTag_p;
		TargetFIXTag_ = TargetFIXTag_p;
		OnBehalfOfFIXTag_ = OnBehalfOfFIXTag_p;
		DeliverToFIXTag_ = DeliverToFIXTag_p;
	  
  	}
};

File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test4/Java/SubID.java
package object;



// Class for tag SubID
public class SubID {

  String SenderFIXTag;
  String TargetFIXTag;
  String OnBehalfOfFIXTag;
  String DeliverToFIXTag;


  // default constructor
  public SubID ( String SenderFIXTag, String TargetFIXTag, String OnBehalfOfFIXTag, String DeliverToFIXTag ) {
    this.SenderFIXTag = SenderFIXTag;
    this.TargetFIXTag = TargetFIXTag;
    this.OnBehalfOfFIXTag = OnBehalfOfFIXTag;
    this.DeliverToFIXTag = DeliverToFIXTag;
  
  }


  public SubID ( ) {   
  
this.SenderFIXTag = "50";
this.TargetFIXTag = "57";
this.OnBehalfOfFIXTag = "116";
this.DeliverToFIXTag = "129";
  
  }
      

} 